microsoft/testsuites/sysbench/sysbenchperf.py

```diff
@@ -0,0 +1,406 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List
+
+from assertpy.assertpy import fail
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+)
+from lisa.messages import TestStatus, send_sub_test_result_message
+from lisa.testsuite import TestResult
+from lisa.tools import Sysbench
+from lisa.util.process import ExecutableResult
+
+
+@dataclass
+class SysbenchTestResult:
+    name: str = ""
+    status: TestStatus = TestStatus.QUEUED
+    message: str = ""
+
+
+@TestSuiteMetadata(
+    area="sysbench",
+    category="performance",
+    description="""
+    This test suite is for executing the sysbench tests
+    """,
+)
+class SysbenchPerfTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Runs Sysbench test for cpu
+        """,
+        priority=3,
+    )
+    def verify_sysbench_cpu_perf(
```

Comment:
The test case names should be like `perf_sysbench_cpu`.

Reply:
Done for all 3 testcases.